.github/workflows/release.yaml

```diff
@@ -58,3 +71,25 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: |
             ${{ steps.meta.outputs.labels }}
+
+      - name: Run Trivy vulnerability scanner on api
+        if: steps.semrel.outputs.version != ''
+        uses: aquasecurity/trivy-action@0.19.0
```

Comment:
is there a reason you are not just calling the trivy-scan workflow?

Reply:
IIRC i had the same problem in https://github.com/adfinis/outdated and it needed the digest, additionally IIRC it would have to be an action to be callable as step

Reply:
If we can move this to an action that would be great, i briefly looked at composite actions and they look promising

Reply:
i think i also asked the same q in outdated :)

i guess this is ok for now and we can for sure investigate the possibility of an action in the future